### ISSUE TITLE
opening_brace violation issue

### ISSUE THREAD

**Issue by tungfam** [realm/SwiftLint#1811]
Running v 0.22.0 and checked the created issues.

### Bug Report

Here is my code:
<img width="1115" alt="screen shot 2017-08-31 at 12 57 47 pm" src="https://user-images.githubusercontent.com/17186720/29917886-02ad8a02-8e4c-11e7-9d34-ff2d717671f7.png">

The warning says:
> Opening Brace Spacing Violation: Opening braces should be preceded by a single space and on the same line as the declaration. (opening_brace)

Adding disable rule:
<img width="1134" alt="screen shot 2017-08-31 at 1 00 01 pm" src="https://user-images.githubusercontent.com/17186720/29918092-c243f414-8e4c-11e7-81e1-c6da0cd95838.png">

And got this issue:
> Superfluous Disable Command Violation: SwiftLint rule 'opening_brace' did not trigger a violation in the disabled region. Please remove the disable command. (superfluous_disable_command)

**Issue: There is a conflict of warning and error.**

PS: though, disabling using `//swiftlint:disable opening_brace` works. but if use `:this` - having issues.

**Comment by marcelofabri** [comment]
The issue is that `opening_brace` rule currently does not do a very good job on reporting the location.

I'd suggest you to use one of the alternatives:
```swift
func foo()
{ // swiftlint:disable:previous opening_brace

}

// swiftlint:disable:next opening_brace
func foo()
{

}
```

Alternatively, you can disable `superfluous_disable_command` as you'd with any rule: in your configuration or with comments.

I think we should keep this open to track improving `opening_brace` violation locations.

**Comment by tungfam** [comment]
Yeah, I made a workaround. Thanks!
